mod/cp_notifications/views/default/cp_notifications/site_template.php

```diff
@@ -322,35 +314,44 @@
 		$topic_author = get_entity($vars['cp_topic']->owner_guid);
 
 		$vars['cp_topic']->title2 = gc_explode_translation($vars['cp_topic']->title,'fr');
-		
-		if($vars['cp_topic']->getSubtype == 'answer'){
-		$question_guid = $vars['cp_topic']->getContainerGUID();
-		$answer_entity = get_entity($question_guid);
-		$title_answer = gc_explode_translation($answer_entity->title,'en');
-		$title_answer2 = gc_explode_translation($answer_entity->title,'fr');
-}
 
-		if ($vars['cp_topic']->getSubtype == 'answer')
-			$cp_notify_msg_title_en = elgg_echo('cp_notify:body_new_content:title2',array($topic_author->getURL().'?utm_source=notification&utm_medium=email', $topic_author->username, cp_translate_subtype($vars['cp_topic']->getSubtype()), $vars['cp_topic']->getURL().'?utm_source=notification&utm_medium=email', $title_answer),'en');
+		if ($vars['cp_topic']->getSubtype() == 'answer') {
+			$question_guid = $vars['cp_topic']->getContainerGUID();
+			$answer_entity = get_entity($question_guid);
+			$title_answer = gc_explode_translation($answer_entity->title,'en');
+			$title_answer2 = gc_explode_translation($answer_entity->title,'fr');
+		}
+
+
+		if ($vars['cp_topic']->getSubtype() == 'answer')
```

Comment:
both conditions are merged, and slight modification in condition for a robust string comparison